### PR TITLE
Facebook user was not really logged out without destroying the session. ...

### PR DESCRIPTION
**Pull request by behcetenuygun**
...Now it does...

### CODE REVIEW

File: Security/Logout/FacebookHandler.php [hunk 1]
```diff
@@ -32,6 +32,7 @@ public function __construct(\BaseFacebook $facebook)
 
     public function logout(Request $request, Response $response, TokenInterface $token)
     {
-        $response->headers->clearCookie('fbsr_'.$this->facebook->getAppId());
+        $this->facebook->destroySession();
+        //$response->headers->clearCookie('fbsr_'.$this->facebook->getAppId());
```
Comment (stof) [inline Security/Logout/FacebookHandler.php:36]: please don't use comments to keep old code. Git is about versionning code, comments are not

Comment (behcetenuygun) [inline Security/Logout/FacebookHandler.php:36]: On 02/23/2012 01:47 PM, Christophe Coevoet wrote:

> > @@ -32,6 +32,7 @@ public function __construct(\BaseFacebook $facebook)
> > 
> > ```
> >   public function logout(Request $request, Response $response, TokenInterface $token)
> >   {
> > ```
> > -        $response->headers->clearCookie('fbsr_'.$this->facebook->getAppId());
> > -        $this->facebook->destroySession();
> > -        //$response->headers->clearCookie('fbsr_'.$this->facebook->getAppId());
> >   please don't use comments to keep old code. Git is about versionning code, comments are not
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/FriendsOfSymfony/FOSFacebookBundle/pull/122/files#r480084
> 
> Don't be mad please, I wont do it again ;D

## 

_Behçet Mutlu_
/Senior PHP Developer, Enuygun.com http://enuygun.com A.Ş./

Merdivenköy Yolu Cad. No:6
D Plaza B Blok Kat 4-5
34750 Ataşehir İstanbul

Tel: 0216 575 0140
Faks: 0216 575 0142

Comment (stof) [inline Security/Logout/FacebookHandler.php:36]: Well, please fix it in the PR so that it can be merged

**Comment by lsmith77** [comment]
@behcetenuygun ping

**Comment by maoueh** [comment]
I think he opened a new PR [#128](https://github.com/FriendsOfSymfony/FOSFacebookBundle/pull/128) with the same fix but without the comment. This PR should be closed.

**Comment by behcetenuygun** [comment]
You are right..

**Pull request closed without merging**